scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,13 +56,24 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader():
-    path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+def get_os_loader(firmware_type=None):
+    osloader = ""
     if 'aarch64' in get_machine_arch():
-        return """
-  <loader readonly='yes' type='pflash'>%s</loader>
-""" % path
-    return ""
+        path = "/usr/share/qemu/aavmf-aarch64-code.bin"
+        osloader = "<loader readonly='yes' type='pflash'>%s</loader>" % path
+    elif 'x86_64' in get_machine_arch() and firmware_type == "uefi":
+        path = "/usr/share/qemu/ovmf-x86_64-ms-code.bin"
+        osloader = "<loader readonly='yes' type='pflash'>%s</loader>" % path
```

Comment:
https://github.com/SUSE-Cloud/automation/pull/1141#discussion_r75468641 is IMHO a nicer way to do this.